Doc/tools/warnings-to-gh-actions.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
+
+"""Convert Sphinx warning messages to GitHub Actions
```

Comment:
```suggestion
"""
Convert Sphinx warning messages to GitHub Actions.
```

Follow PEP 257, the docstring standard